frame/timestamp/src/lib.rs

```diff
@@ -148,7 +148,7 @@ decl_module! {
 		/// `MinimumPeriod`.
 		///
 		/// The dispatch origin for this call must be `Inherent`.
-		#[weight = SimpleDispatchInfo::FixedMandatory(MINIMUM_WEIGHT)]
+		#[weight = (MINIMUM_WEIGHT, frame_support::weights::DispatchClass::Mandatory)]
```

Comment:
Can you just import this above?

If it is too much of a pain, let me know, I am happy to do it :)

Reply:
no problem, I already noted it above as a todo to clean this stuff.